test/functional/data/complexTestData.js

```diff
@@ -1,17 +1,17 @@
 export default [{
   // just small protein loaded by default
   name: '1CRN',
-  formats: ['pdb', 'cif', 'mmtf'],
+  formats: ['pdb', 'cif', 'mmtf', 'xyz'],
   num: {
     atoms: 327,
     bonds: 337,
-    residues: 46,
+    residues: {xyz: 1, default: 46},
     chains: 1,
-    molecules: 1,
-    units: 2,
+    molecules: {xyz: 0, default: 1},
```

Comment:
Please generate one molecule in your parser (include everything there by default).

Reply:
Thanks, change is done